cli/src/cmd/exec.rs

```diff
@@ -350,6 +371,55 @@ impl Cli {
                 print!("{}", diff);
                 Ok(ExitCode::SUCCESS)
             }
+            Subcommand::CallPlugin(identifier, function, input_arg, output) => {
+                let mut input = input_arg.as_bytes().to_vec();
+
+                // if the value is a path to a file
+                if input_arg.to_owned().starts_with("@") {
+                    let path = input_arg
+                        .chars()
+                        .skip(1)
+                        .take(input_arg.len() - 1)
+                        .collect::<String>();
+                    input = tokio::fs::read(Path::new(&path)).await?;
+                }
+
+                let client = Client::new(self.host.as_str())?;
+                let res = client.call_plugin(identifier, function, input).await?;
+
+                if let Some(output) = output {
+                    tokio::fs::write(output, res).await?;
+                } else {
+                    std::io::stdout().write_all(&res);
+                }
+
+                Ok(ExitCode::SUCCESS)
+            }
+            Subcommand::InstallPlugin(identifier, name, location, wasm) => {
+                // TODO: implement location handling
+                if let Some(wasm) = wasm {
+                    let wasm = tokio::fs::read(wasm).await?;
+                    let name = match name {
+                        Some(v) => v,
+                        None => "",
+                    };
+                    let location = match location {
+                        Some(v) => v,
+                        None => "",
+                    };
+                    let client = Client::new(self.host.as_str())?;
+                    let res = client
+                        .install_plugin(identifier, name.to_string(), location.to_string(), wasm)
+                        .await?;
+                    return Ok(ExitCode::SUCCESS);
+                }
+                Err(anyhow!("No wasm provided.  Note: the --location argument is currently not supported.  Please use --wasm instead for now."))
```

Comment:
We can probably re-use `--wasm` for the "location" concept as well.. would want to parse the value into a `url::Url` and if it succeeds, fetch & load that for installation, and if it fails parse as url, we can try it as the wasm file input?

Reply:
Yeah that is a good point.  I was thinking of keeping these two concepts separate in case we wanted to allow folks to specify a location to their wasm, but also take that wasm and store it as a binary blob in Modsurfer's DB as well (this way after the initial download, their instance won't need internet access to utilize the plugin).  I don't think what you're suggesting precludes us from being able to do so though, as long as we have separate `location` and `wasm` columns in the DB. I'll see if I can console the CLI args `--location` and `--wasm` into just `--wasm`

Reply:
ah I see - in my head, the plug-in which is pulled from a `location` is only done at install time, and then is stored in the DB as a blob so there isn't repeated trips to download the module.

Reply:
after re-reading, I think we're saying the same thing lol. so ya, I think it makes sense to consolidate these into the same argument?

Reply:
Yup we are on the same page!

Reply:
This change is implemented. I ended up just implementing the ability to retrieve the plugin's wasm from a remote location as well. The `location` & `wasm` will always be set in the DB, whether it's a URL or file path.